tests/lib/edge_impulse/CMakeLists.txt

```diff
@@ -35,3 +35,4 @@ target_sources(app PRIVATE src/main.cpp)
 # Test uses ei_test_params.h file from edge_impuse_zip directory to verify if
 # ei_wrapper properly forwards the data between application and EI library.
 target_include_directories(app PRIVATE src/edge_impulse_zip/)
+target_include_directories(app PRIVATE src/edge_impulse_zip/edge-impulse-sdk/)
```

Comment:
Consider placing the header file in `src/edge_impulse_zip/`. The `edge-impulse-sdk` subdirectory contains source files for Edge Impulse library and it should not be directly included by the test (test is expected to access the EI library only via EI wrapper API - except for the newly introduced clearing API).